ogr/ogrsf_frmts/openfilegdb/ogropenfilegdblayer_write.cpp

```diff
@@ -2759,7 +2759,8 @@ void OGROpenFileGDBLayer::RefreshXMLDefinitionInMemory()
                 pszShapeType = "esriGeometryMultipoint";
                 break;
             case FGTGT_LINE:
-                pszShapeType = "esriGeometryLine";
+                pszShapeType = m_bArcGISPro32OrLater ? "esriGeometryPolyline"
+                                                     : "esriGeometryLine";
```

Comment:
Actually checking with the FileGDB driver, we have always written "esriGeometryPolyline", so it doesn't seem to have compatibility issue, and we can just unconditionally do:

```suggestion
                pszShapeType = "esriGeometryPolyline";
```